README.md

```diff
@@ -1,11 +1,11 @@
-[![Travis branch](https://img.shields.io/travis/TechnologyAdvice/DevLab/master.svg)](https://travis-ci.org/TechnologyAdvice/DevLab)
-[![codecov](https://codecov.io/gh/TechnologyAdvice/DevLab/branch/master/graph/badge.svg)](https://codecov.io/gh/TechnologyAdvice/DevLab)
+[![Travis branch](https://img.shields.io/travis/Binci/binci/master.svg)](https://travis-ci.org/Binci/binci)
+[![codecov](https://codecov.io/gh/Binci/binci/branch/master/graph/badge.svg)](https://codecov.io/gh/Binci/binci)
 
-<img src="./devlab.png" align="right">
+<img src="./binci.png" align="right">
 
-# DevLab
+# Binci
 
-DevLab is a utility that allows you to easily containerize your development
+Binci is a utility that allows you to easily containerize your development
 workflow using Docker. Simply put; it's like having a cleanroom for all of your
 development processes which contains services (like databases) without needing

```

Comment:
Then this should be changed to the grammatically correct option `contain` as that rolls off the tongue better 😜